src/test/java/com/wikia/webdriver/common/dataprovider/CrossWikiSearchProvider.java

```diff
@@ -19,19 +17,11 @@ private CrossWikiSearchProvider() {
 
   @DataProvider
   public static final Object[][] getExactMatchQueries() {
-    return new Object[][]{
-        {
-            "call of duty", "Call of Duty Wiki", "VIDEO GAMES"
-        }, {
-            "call-of-duty", "Call of Duty Wiki", "VIDEO GAMES"
-        }, {
-            "call_of_duty", "Call of Duty Wiki", "VIDEO GAMES"
-        }, {
-            "callofduty", "Call of Duty Wiki", "VIDEO GAMES"
-        }, {
-            "cod", "Call of Duty Wiki", "VIDEO GAMES"
-        },
-    };
+    return new Object[][] { {"call of duty", "Call of Duty Wiki", "VIDEO GAMES"},
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Call of Duty Wiki" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "VIDEO GAMES" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)